webapp/tsconfig.json

```diff
@@ -2,17 +2,18 @@
   "compileOnSave": false,
   "compilerOptions": {
     "baseUrl": "./",
+    "downlevelIteration": true,
```

Comment:
From [the release notes from typescript](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-3.html#new---downleveliteration), if you use `target >= es2015`, you don't need it.

Reply:
An automatic change done by the Angular upgrade script, along with the change of target. :-/